src/edu/gatech/cs2340/trydent/gui/TrydentDropdown.java

```diff
@@ -0,0 +1,67 @@
+package edu.gatech.cs2340.trydent.gui;
+
+import javafx.beans.value.ChangeListener;
+import javafx.collections.ObservableList;
+import javafx.scene.control.ChoiceBox;
+
+/**
+ * A basic dropdown box
+ * @author sparky
+ *
+ */
+@SuppressWarnings("rawtypes")
```

Comment:
ChoiceBox is sort of strange in that it actually supports adding mixtures of objects.
It actually can take a Converter object which tells it how to display items, so I did this to maintain that flexibility.